src/components/ContactsList/Contacts/ContactIdentity.jsx

```diff
@@ -1,27 +1,32 @@
 import React from 'react'
-import PropTypes from 'prop-types'
 
 import { Avatar } from 'cozy-ui/transpiled/react/Avatar'
+import { models } from 'cozy-client'
+
+import { fullContactPropTypes } from '../../ContactPropTypes'
 import ContactName from './ContactName'
-import contactPropTypes from '../../ContactPropTypes'
-import { getInitials } from '../../../helpers/contacts'
+
+const { getDisplayName, getInitials } = models.contact
 
 const MyselfMarker = (props, { t }) => (
   <span className="contact-myself">({t('me')})</span>
 )
-const ContactIdentity = ({ name, myself }) => (
-  <div className="contact-identity">
-    <Avatar text={getInitials(name).toUpperCase()} size="small" />
-    <ContactName firstname={name.givenName} lastname={name.familyName} />
-    {myself && <MyselfMarker />}
-  </div>
-)
-ContactIdentity.propTypes = {
-  name: contactPropTypes.name.isRequired,
-  myself: PropTypes.bool
+
+const ContactIdentity = ({ contact }) => {
+  const name = contact.name || {}
+  const displayName = getDisplayName(contact) || undefined
+  const isMyself = contact.metadata ? !!contact.metadata.me : false
+
+  return (
+    <div className="contact-identity">
+      <Avatar text={getInitials(contact)} size="small" />
+      <ContactName displayName={displayName} lastname={name.familyName} />
```

Comment:
We should avoid to change the name of a prop, we should keep familyName, no? 

Also, I'm wondering if `ContactName` should not receive the full `contact` object and call displayName() etc by itself.

Reply:
Ok for `familyName`.

For `ContactName` I think it depends if we want to have a dumb component or not. For now the only treatment in it is done for styling, not to treat datas.

Reply:
I don't know, to me if the component is named `ContactX` it should handle a contact object. But it's a nit ;)